pkgs/top-level/python-packages.nix

```diff
@@ -10380,6 +10380,8 @@ in {
 
   tatsu = callPackage ../development/python-modules/tatsu { };
 
+  taxi = callPackage ../development/python-modules/taxi { };
```

Comment:
If the package is a cli it does not need to be added to pythonPackages which allows us more freedom with overrides and options.

Reply:
I'm not sure what would be the best way to deal with this. Because taxi is a cli program and a library. The backends it supports call the taxi library to interface later with taxi. Here is an example backend: https://github.com/liip/taxi-zebra which imports taxi as a library for example.

However a backend needs to be present in the pythonpath of the taxi cli for it to work. because the cli has a list of backend it can try to call.

I'm not sure what would be the best way to go about this.

Reply:
If it is used as a library, too then this is fine.